docs/IE_PLUGIN_DG/Intro.md

```diff
@@ -61,5 +62,5 @@ Detailed guides
 API References
 -----------------------
 
-* [Inference Engine Plugin API](groupie_dev_api.html)
-* [Inference Engine Transformation API](groupie_transformation_api.html)
+* [Inference Engine Plugin API](@ref ie_dev_api)
+* [Inference Engine Transformation API](@ref ie_transformation_api)
```

Comment:
OpenVINO Runtime?

Reply:
have dedicated task CVS-80084 to update Plugin development guide